test/unit/storage/conversion/QuadToRdfConverter.test.ts

```diff
@@ -126,6 +127,27 @@ test:s dc:modified test:o.
     ]
   }
 ]
+`,
+    );
+  });
+
+  it('converts quads in the ResponseMetadata graph to the default graph (in Turtle)..', async(): Promise<void> => {
```

Comment:
```suggestion
  it('converts quads in the ResponseMetadata graph to the default graph.', async(): Promise<void> => {
```
It does this for all serializations as far as I can tell?